packages/checkout/sdk-sample-app/src/components/Cancel.tsx

```diff
@@ -69,6 +82,22 @@ export default function Cancel({ checkout, provider }: CancelProps) {
         )}
       </FormControl>
       <br />
+      <FormControl>
+        <FormControl.Label>Use gasless cancel</FormControl.Label>
+        <Checkbox
+          checked={useGaslessCancel}
+          onClick={updateUseGasless}
```

Comment:
Something was logging a warning here that the `onChange` wasn't set.. although this works fine is it work switching the `onClick` to `onChange` ?
<img width="1220" alt="Screenshot 2024-02-08 at 2 52 14 pm" src="https://github.com/immutable/ts-immutable-sdk/assets/1617003/50ea22fe-f88a-4033-a251-28e74f307151">